web3/providers/eth_tester/defaults.py

```diff
@@ -178,6 +178,7 @@ def personal_send_transaction(eth_tester, params):
         ),
         'getBalance': call_eth_tester('get_balance'),
         'getStorageAt': not_implemented,
+        'getProof': not_implemented,
```

Comment:
Of course, it would be ideal to get eth-tester to support this. Seems out of scope, but once it's back in, it will enable more straightforward testing options back here in web3.py.